src/selectors/shelf.test.ts

```diff
@@ -21,6 +22,9 @@ describe('selectors/shelf', () => {
           config: {
             ...DEFAULT_VOYAGER_CONFIG
           },
+          log: {
```

Comment:
Why don't we just make `persistent: DEFAULT_PERSISTENT_STATE`?

Seems like we're not adding anything here anyway.